src/base64.js

```diff
@@ -1,9 +1,9 @@
 'use strict';
 
 export function base64(i) {
-  return (new Buffer(i, 'ascii')).toString('base64');
+  return (new Buffer.from(i, 'ascii')).toString('base64');
 }
 
 export function unbase64(i) {
-  return (new Buffer(i, 'base64')).toString('ascii');
+  return (new Buffer.from(i, 'base64')).toString('ascii');
```

Comment:
As @intellix mentions `new` can be removed

Reply:
Yeah, that was a whoops on my part.